src/components/form/Form.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useForm } from 'react-hook-form';
+
+const Form = props => {
+  const {
+    defaultValues,
+    children,
+    onSubmit,
+    validationResolver,
+    ...rest
+  } = props;
+
+  const methods = useForm({ defaultValues, validationResolver });
+  const { handleSubmit } = methods;
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)} {...rest}>
+      {Array.isArray(children)
```

Comment:
this ternary is pretty complex. can we break it up?

Reply:
☝️ Yeah, I think that's what I was referring to in regards to trying it more readable. If it's copied from  https://react-hook-form.com/advanced-usage#SmartFormComponent then maybe we should make that comment at least.

Reply:
Makes sense -- maybe adding a comment that this came from https://react-hook-form.com/advanced-usage#SmartFormComponent  will fend off future questions about why this bit was written this wya.

Reply:
Code refactored: https://github.com/codebuddies/frontend/pull/105/commits/16537363bb00d03c5b3112c16f531ccca979b309